Dockerfile_tesseract_training

```diff
@@ -1,31 +1,33 @@
 
-# Run this dockerfile using the command below from repo root
-# docker-compose -f docker.dev.yml up -d
+# Run this dockerfile using the command below from repo root for model training
+# docker-compose -f docker.dev.yml up -d train
```

Comment:
this should have a --build flag on it so that when the dev fires up the container it uses the newest version of the code. Without that flag, if a previous build exists docker will just use that since its cached. That means that the if the user changes code and reruns this command, nothing will change. So ideallyyou want 

`docker-compose -f docker.dev.yml up --build -d train`

☝️ but I also recommend renaming `docker.dev.yml` to `docker-compose.yml` as that is more standardized practice